SplunkRumWorkspace/SplunkRum/SplunkRum.xcodeproj/project.pbxproj

```diff
@@ -612,6 +612,7 @@
 				SKIP_INSTALL = YES;
 				SUPPORTS_MACCATALYST = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_PRECOMPILE_BRIDGING_HEADER = NO;
```

Comment:
Is this addition necessary? Same for the other `SWIFT_PRECOMPILE_BRIDGING_HEADER` line

Reply:
sorry its not necessary, i will revert this.